common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -181,69 +190,69 @@ function () {
         return false;
     }
 
-    function _keyDownHandler(event) {
-        var KEY = $.ui.keyCode,
-            keyCode = event.keyCode,
-            target = $(event.currentTarget),
-            index;
-
-        if (target.is('a.a11y-menu-item-link')) {
-
-            index = target.parent().index();
-
-            switch (keyCode) {
-                // Scroll up menu, wrapping at the top. Keep menu open.
-                case KEY.UP:
-                    _previousMenuItemLink(this.menuItemsLinks, index).focus();
-                    break;
-                // Scroll down  menu, wrapping at the bottom. Keep menu
-                // open.
-                case KEY.DOWN:
-                    _nextMenuItemLink(this.menuItemsLinks, index).focus();
-                    break;
-                // Close menu.
-                case KEY.TAB:
-                    _closeMenu(this);
-                    // TODO
-                    // What has to happen here? In speed menu, tabbing backward
-                    // will give focus to Play/Pause button and tabbing
-                    // forward to Volume button.
-                    break;
-                // Close menu, give focus to button and change
-                // file type.
-                case KEY.ENTER:
-                case KEY.SPACE:
-                    this.button.focus();
-                    this.changeFileType.call(this, event);
-                    _closeMenu(this);
-                    break;
-                // Close menu and give focus to speed control.
-                case KEY.ESCAPE:
-                    _closeMenu(this);
-                    this.button.focus();
-                    break;
-            }
-            return false;
-        }
-        else {
-            switch(keyCode) {
-                // Open menu and focus on last element of list above it.
-                case KEY.ENTER:
-                case KEY.SPACE:
-                case KEY.UP:
-                    _openMenu(this);
-                    this.menuItemsLinks.last().focus();
-                    break;
-                // Close menu.
-                case KEY.ESCAPE:
-                    _closeMenu(this);
-                    break;
-            }
-            // We do not stop propagation and default behavior on a TAB
-            // keypress.
-            return event.keyCode === KEY.TAB;
-        }
-    }
+//     function _keyDownHandler(event) {
```

Comment:
I removed keyboard (up/down/etc) from this menu. @cptvitamin said it was okay so long as we allowed for the normal keyboard navigation, both with and without a screen reader. Kept this here in case we decide to use it.